presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -21,5 +22,6 @@ data class EventScreenState(
     val endDate: ZonedDateTime = ZonedDateTime.now(ZoneId.systemDefault()),
     val isStartDateTime: Boolean = false,
     val shouldOpenDropdown: Boolean = false,
-    val alarmReminderItem: AlarmReminderItem = AlarmReminderItem.TEN_MINUTES
+    val alarmReminderItem: AlarmReminderItem = AlarmReminderItem.TEN_MINUTES,
+    val listOfAttendee: List<Attendee> = listOf()
```

Comment:
list of all attendees that have been added to the event

Reply:
That's what the name says 😁